translation/file_loader_test.go

```diff
@@ -2,34 +2,52 @@ package translation
 
 import (
 	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
 
+	"github.com/goravel/framework/mocks/foundation"
 	"github.com/goravel/framework/support/env"
 	"github.com/goravel/framework/support/file"
 )
 
 type FileLoaderTestSuite struct {
 	suite.Suite
+	app        *foundation.Application
+	executable string
 }
 
 func TestFileLoaderTestSuite(t *testing.T) {
 	assert.Nil(t, file.Create("lang/en/test.json", `{"foo": "bar", "baz": {"foo": "bar"}}`))
 	assert.Nil(t, file.Create("lang/en/another/test.json", `{"foo": "backagebar", "baz": "backagesplash"}`))
 	assert.Nil(t, file.Create("lang/another/en/test.json", `{"foo": "backagebar", "baz": "backagesplash"}`))
 	assert.Nil(t, file.Create("lang/en/invalid/test.json", `{"foo": "bar",}`))
-	// We should adapt this situation.
 	assert.Nil(t, file.Create("lang/cn.json", `{"foo": "bar", "baz": {"foo": "bar"}}`))
 	restrictedFilePath := "lang/en/restricted/test.json"
 	assert.Nil(t, file.Create(restrictedFilePath, `{"foo": "restricted"}`))
 	assert.Nil(t, os.Chmod(restrictedFilePath, 0000))
+
+	// path2
+	executable, err := executablePath()
+	assert.NoError(t, err)
+	assert.Nil(t, file.Create(filepath.Join(executable, "lang/en/test.json"), `{"foo": "bar", "baz": {"foo": "bar"}}`))
```

Comment:
Does the file cover the file of line 23? We can create another file for this  case.

Reply:
> Does the file cover the file of line 23? We can create another file for this case.

No, it doesn't.